extended-utxo-spec/extended-utxo-specification.tex

```diff
@@ -573,14 +609,17 @@ \subsubsection{The $\ptx$ type}
 which produces the relevant $\ptx$ for a transaction $t$ and an input
 $i$, and a function
 $$
-\sigma: \ptx \rightarrow \script
+\delta: \ptx \rightarrow \Data
 $$
-which translates a $\ptx$ object into a script containing exactly the
-same information, encoded in some standard way enabling it to be accessed
-by a validator script .  Assuming we have an appropriate hashing
-function, it is straightforward to define $\tau$.  The function
-$\sigma$ is implementation-dependent and we will not discuss it
-further.
+which translates a $\ptx$ object into a \Data{} value containing
+exactly the same information, encoded in some standard way enabling it
+to be accessed by a validator script.  Assuming we have an
+appropriate hashing function, it is straightforward to define $\tau$.
+The function $\delta$ is implementation-dependent and we will not
+discuss it further.
+
+\medskip\todokwxm{But with the advent of \Data{} we \textit{can}
+  specify $\delta$.  Maybe we should?}
```

Comment:
Yes, we should. Although it will be tedious, since `PendingTx` is pretty big. I think it's fine not to do it for now, and it would be an appendix anyway.